src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+    class="search" 
+    action="/search" 
+    method="post" 
+    data-qa="keypress"
+    >
+      <img 
+      src="./images/Search.svg" 
+      alt="pic" 
+      class="search-icon"
+      >
+      <label>
+        <input 
+        data-qa="big"
+        type="search" 
+        class="query" 
```

Comment:
create two classes for both forms, one common, one different to prevent styles duplicating